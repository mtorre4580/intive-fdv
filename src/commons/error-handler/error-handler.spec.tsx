import { mount, shallow } from 'enzyme';
import ErrorBoundary from './error-handler';
import  React from 'react';
import renderer from 'react-test-renderer';

describe('<ErrorBoundary />' , () => {
   
    it('should render snapshot', () => {
        const component = renderer.create(<ErrorBoundary><div>content mock</div></ErrorBoundary>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

})