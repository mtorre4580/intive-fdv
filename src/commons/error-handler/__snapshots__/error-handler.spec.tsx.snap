// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundary /> should render snapshot 1`] = `
<div>
  content mock
</div>
`;
